.github/pull_request_template.md

```diff
@@ -5,17 +5,6 @@
 
 --- Merge policy ---
 
-- [ ] Travis CI PASS
-- [ ] `*-aws-runtest` PASS
-- [ ] `*-azure-runtest` PASS
-- [ ] `*-images-runtest` PASS
-- [ ] `*-openstack-runtest` PASS
-- [ ] `*-vmware-runtest` PASS
+- [ ] `Required` CI checks PASS
 - [ ] For `rhel8-*` and `rhel7-*` branches commit log references an approved
   bug in Bugzilla. Do not merge if the bug doesn't have the 3 ACKs set to `+`!
-
```

Comment:
With the new CI system are there any commands we can use in the PR? Or do we have to re-trigger them using cmdline tools?

Reply:
@martinpitt, @larskarlitski ^^^

@bcl I know only about cli tools for now.

Reply:
Tests can be retriggered only with CLI tools right now.